code/modules/events/disease_outbreak.dm

```diff
@@ -301,21 +324,22 @@
  * If the virus is airborne, also don't hide it.
  */
 /datum/disease/advance/random/event/AssignProperties()
-	var/transmissibility = rand(1, 100)
 
-	addtimer(CALLBACK(src, PROC_REF(MakeVisible)), ((ADV_ANNOUNCE_DELAY * 2) - 10) SECONDS)
+	addtimer(CALLBACK(src, PROC_REF(makeVisible)), ((ADV_ANNOUNCE_DELAY * 2) - 10) SECONDS)
 
 	if(length(properties))
 		spreading_modifier = max(CEILING(0.4 * properties["transmittable"], 1), 1)
 		cure_chance = clamp(7.5 - (0.5 * properties["resistance"]), 5, 10) // Can be between 5 and 10
-		stage_prob = max(0.5 * properties["stage_rate"], 1)
+		stage_prob = max(0.4 * properties["stage_rate"], 1)
 		SetSeverity(properties["severity"])
 		visibility_flags |= HIDDEN_SCANNER
-		switch(severity)
-			if("Dangerous", "BIOHAZARD")
-				visibility_flags &= ~HIDDEN_SCANNER
 
-		GenerateCure(properties)
+		if(severity == "Dangerous" || severity == "BIOHAZARD")
+			visibility_flags &= ~HIDDEN_SCANNER
+			SetSpread(DISEASE_SPREAD_CONTACT_SKIN)
+			return
```

Comment:
We return without calling `GenerateCure()`, intended?

Reply:
No, it should be calling GenerateCure, but not bother with the transmissibility roll and if chain